personal-portfolio/js/script.js

```diff
@@ -0,0 +1,22 @@
+const menu_btn = document.querySelector('.hamburger');
+const mobile_menu = document.querySelector('.mobile-nav');
+
+menu_btn.addEventListener('click', function () {
+  menu_btn.classList.toggle('is-active');
+  mobile_menu.classList.toggle('is-active');
+});

```

Comment:
- Hello nice job on fixing the errors mentioned by the previous TSE, but you are still on an issue.

- The close button does not disappear after clicking on it to close the menu we should see the hamburger icon instead
![porfolio](https://user-images.githubusercontent.com/61562626/128366608-016d2cb1-e2a0-4f45-a89c-748861b3e166.gif)